src/components/Fetcher/index.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Spinner from 'react-spinkit'
+import { connect } from 'react-redux'
+
+import LANGUAGE_FETCHER from 'endpoints/language'
+import PAGE_ENDPOINT from 'endpoints/page'
+import DISCLAIMER_FETCHER from 'endpoints/disclaimer'
+import LOCATION_FETCHER from 'endpoints/location'
```

Comment:
They are used on the bottom of the file. This will change in an other CR. We need to merge the endpoins/ and Fetcher/ package. The Fetchers belong to the endpoint API and are not a standalone component.

Please take a look at this: https://www.npmjs.com/package/react-redux-fetch
I think we should not remove our code and just use this package because it is not used much and we're fucked when they stop supporting it. But we maybe can steal some ideas.